walkthroughs/howto-http-retries/README.md

```diff
@@ -33,31 +34,28 @@ We will use this endpoint in curl to pass header values to the header _statuscod
 
 ## Update Route
 
-1. Check the current route configuration by calling describe-route. For instance, to describe the blue route (notice the retry policy in the spec):
+1. Check the current route configuration by calling describe-route. For instance, to describe the blue route:
     ```
     aws appmesh describe-route --mesh-name howto-http-retries --virtual-router-name color-router --route-name color-route-blue
     ```
     
-2. Use curl to send a request to the frontend service without any headers. The frontend service will route the request to the backend blue service.
-The blue service will return the color blue and you can observe the 200 response code in the logs found in 
-_Cloudwatch -> Logs -> howto-http-retries-log-group -> blue/app/task_id_. Filter out the ping requests by searching _-"ping"_
+2. Use curl to send a request to the frontend service. The frontend service will route the request to the backend blue service.
+The blue service will experience a failure and return a 503. 
     ```
     curl <endpoint>
     ``` 
     
-3. Use curl to send a request to your frontend service but this time add the header below to see retry policy in place. In the cloudformation template we set the
-retry limit to 4 so you should see 5 total failures in your logs. One for the initial request and then 4 retries. Search the logs in the same way, filter out the pings.  
-    ```
-    curl -H "statuscode-header: 503" <endpoint> -v    
-    ```
-    
-4. Feel free to update the route by updating the json file in this package. For instance you can update it to retry 2 times instead of 4:
+3. Now lets introduce retries and watch our service return a 200 by returning "blue". The backend service will return a 200 only after 1 second has passed since the initial request from the frontend service. Update your route configuration to include retries by running the following command:
```

Comment:
super nit: typo `let's`

My general concern with this approach is that it's sort conflating the frontend request with the backend since we're doing this contrived thing with the time header. Realistically the backend would see these as independent requests. And I think this could be confusing along with timeout (https://github.com/aws/aws-app-mesh-roadmap/issues/53) when that becomes available. Thoughts?